whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapObjectMapperTest.java

```diff
@@ -187,6 +182,57 @@ public void autnum() {
         assertThat(result.getPort43(), is("whois.ripe.net"));
     }
 
+    @Test
+    public void asBlock() {
+        final Autnum result = (Autnum) map((RpslObject.parse("" +
+                "as-block:       AS100-AS200\n" +
```

Comment:
as-block objects in the database always include spaces, e.g. "AS100 - AS200", can you fix the test data?